salt-mode.el

```diff
@@ -54,12 +54,23 @@
 ;; C-c % %
 ;;   for {# and {% as well.
 
+;; In-Emacs documentation: If you have Salt installed on the same
```

Comment:
Is `In-Emacs documentation` a special header that emacs will pick up? I couldn't see this comment when running `describe-mode`.